### ISSUE TITLE
scripts can't be run on linux due to no winreg for unix

### ISSUE THREAD

**Issue by shadow-absorber** [farpeter98/df-combat-clicker#1]
there is no winreg module for unix based systems due to requirements for windows registry making the reckoning script not work... dunno if you are still active and thinking about updating this or if it is abandon ware but just wanted to let you know

**Comment by farpeter98** [comment]
Yeah, it's more of a botched together project quickly mashed together to work on a very specific setup, using flash projector on wsl running on a windows machine, so as to have the whole thing render via opengl instead of native windows handles, thus making clicks more trivial, only requiring the main window handle instead of the each individual controls.

The win32 api calls are somewhat contained so the implementations could be replaced to have clicks, window handles, and screenshots work on unix too, but personally I won't be updating it, feel free to use it however you see fit.

Edit: the first script used pyautogui, which I abandoned cause I couldn't make it work without it using the actual cursor, so it always required the window to be on top, and the first solution I found was just to use native windows calls.
